corda-common-test/src/main/java/ch/cordalo/corda/common/test/CordaTestNetwork.java

```diff
@@ -24,32 +25,26 @@
 public class CordaTestNetwork {
 
     private final List<Class<? extends FlowLogic>> responderClasses;
-    private MockNetwork network;
-    private final List<TestCordapp> testCordapps;
+    private final MockNetwork network;
+    private final List<TestCordapp> testCorDApps;
     private final List<String> testPackageNames;
     private final boolean withNodes;
     private final Map<String, CordaNodeEnvironment> nodes = new HashMap();
 
     public CordaTestNetwork(boolean withNodes, List<String> testPackageNames, List<Class<? extends FlowLogic>> responderClasses) {
         this.testPackageNames = testPackageNames;
-        this.testCordapps = testPackageNames.stream().map(x -> TestCordapp.findCordapp(x)).collect(Collectors.toList());
+        this.testCorDApps = testPackageNames.stream().map(x -> TestCordapp.findCordapp(x)).collect(Collectors.toList());
         this.withNodes = withNodes;
         this.responderClasses = responderClasses;
-        this.createNetwork();
+        if (this.withNodes) {
+            network = new MockNetwork(new MockNetworkParameters(testCorDApps));
+        } else {
+            network = null;
+        }
     }
 
     public CordaTestNetwork(boolean withNodes, List<String> testPackageNames, Class responderMainClass) {
-        this.testPackageNames = testPackageNames;
-        this.testCordapps = testPackageNames.stream().map(x -> TestCordapp.findCordapp(x)).collect(Collectors.toList());
-        this.withNodes = withNodes;
-        this.responderClasses = FlowTestSupporter.findResponderClasses(responderMainClass);
-        this.createNetwork();
-    }
-
-    private void createNetwork() {
-        if (this.withNodes) {
-            network = new MockNetwork(new MockNetworkParameters(testCordapps));
-        }
+        this(withNodes, testPackageNames, Arrays.asList(responderMainClass));
```

Comment:
This is wrong. 
You have to search all static innerclasses to find all that are responders. 
And add the to the List.

Reply:
this was the code before i just remove the duplicated code by calling other constructor, no code change